demo/app/app.component.html

```diff
@@ -1,12 +1,4 @@
 <layout-header [version]="version"></layout-header>
-<div class="container">
```

Comment:
Removed because it does not add any value for user and takes a lot of space. Logo with name is already placed in header.